src/quo2/components/tags/tag.cljs

```diff
@@ -59,22 +59,36 @@
       label])])
 
 (defn tag
+  "opts
+   {:type         :icon/:emoji/:label
+    :size         32/24
+    :on-press     fn
+    :blurred      true/false 
+    :resource     icon/image
+    :labelled     true/false
+    :disabled     true/false}
```

Comment:
Can we indicate it's a boolean by adding a question mark at the end
`disabled` -> `disabled?`